firefox-ios/Client/Frontend/Components/ContentContainer.swift

```diff
@@ -65,6 +65,8 @@ class ContentContainer: UIView {
     ///
     /// - Parameter content: The content to update
     func update(content: ContentContainable) {
+        // If the content type is the same as the current then return early
+        guard content != type else { return }
```

Comment:
@OrlaM This fails compilation with `Cannot convert value of type 'any ContentContainable' to expected argument type 'ContentType'`

Reply:
Possibly meant this? 

```suggestion
        guard content.contentType != type else { return }
```